lua/nvim-treesitter/fold.lua

```diff
@@ -0,0 +1,32 @@
+local api = vim.api
+local parsers = require'nvim-treesitter.parsers'
+
+local M = {}
+
+function M.get_fold_indic(lnum)
+  if parsers.has_parser() and lnum then
```

Comment:
`if not parser.has_parser() or not lnum then return '0' end`
then you wouldn't need to have to put everything else in the else ;)

Reply:
lgtm to me otherwise, nice job :)